packages/block-editor/src/components/inspector-popover-header/index.js

```diff
@@ -45,8 +45,8 @@ export default function InspectorPopoverHeader( {
 				) ) }
 				{ onClose && (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
The `__next40pxDefaultSize` prop is not necessary when the `size` prop is specified

```suggestion
```